spec/factories/competitions.rb

```diff
@@ -101,6 +101,15 @@
 
     registration_version { :v3 }
 
+    trait :auto_accept do
+      use_wca_registration { true }
+      auto_accept_registrations { true }
+    end
+
+    trait :allow_self_delete do
+      competitor_can_cancel { :always }
+    end
```

Comment:
Where is this seemingly unrelated trait coming from?

Reply:
Removed - it is only relevant when API tests get added